helm/botkube/values.yaml

```diff
@@ -427,6 +427,14 @@ executors:
         helmConfigDir: "/tmp/helm/"
         # -- Location for storing cached files. Must be under the Helm config directory.
         helmCacheDir: "/tmp/helm/.cache"
+      context:
+        defaultNamespace: "botkube"
+        rbac:
+          group:
+            type: Static
+            prefix: ""
+            static:
+              value: ["botkube-plugins-read-only"]
```

Comment:
1. Please describe the properties with `# -- ` comments which will generate docs for them.
2. Also, please consider defining static user mapping as well. I know we'll fallback to `default` user, but maybe it would be better to also document this as a part of Helm chart values 🤔

Reply:
As we recently discussed (in a different PR) naming - noun plural vs singular - I'm thinking the same about `value`.
I'm more inclined towards the plural form or something like `references`. What do you think?

Reply:
`references` sounds to me like it should be defined somewhere in the config 🤔 maybe `values` then? (only for group mapping)